frontend/src/components/passphrase-modal/passphrase-modal.tsx

```diff
@@ -28,12 +27,12 @@ interface FormFields {
 
 export function PassphraseModal() {
   const [loading, setLoading] = useState(false)
-  const { state, dispatch } = useGlobal()
+  const { state, actions, dispatch } = useGlobal()
 
   // Register handler.open to open the passphrase modal
   useEffect(() => {
     handler.open = () => {
-      dispatch(setPassphraseModalAction(true))
+      dispatch(actions.setPassphraseModalAction(true))
     }
   }, [dispatch])
```

Comment:
chore (blocking): missing actions in deps array here

Reply:
fixed